src/main/resources/assets/cyclopscore/lang/en_US.lang

```diff
@@ -31,3 +31,47 @@ multiblock.cyclopscore.error.hollow.air=Position %s must be hollow.
 
 # Vanilla additions
 tile.air.name=Air
+
+# Config Screens
+config.cyclopscore=configuration
+
+## Config categories
+config.cyclopscore.item=Item settings
+config.cyclopscore.item.tooltip=Change settings of items
```

Comment:
Currently these tooltips don't really give any useful additional information. If there's nothing special about a category, I would remove the tooltip.